.buildkite/scripts/e2e.sh

```diff
@@ -6,4 +6,8 @@ set -eu
 TEST_MARKER=${TEST_MARKER:-regression}
 echo "TEST_ENVIRONMENT_NAME: ${TEST_ENVIRONMENT_NAME}; TEST_MARKER: ${TEST_MARKER}"
 
-docker run --rm -it $(env | cut -f1 -d= | sed 's/^/-e /') -v "$PWD/e2e:/virtru/e2e" -v "$PWD/e2e/reports:/virtru/reports" -v "$PWD/.buildkite/scripts:/virtru/e2e/scripts"  virtru/automated-test:latest pytest -v -s -n2 e2e -m $TEST_MARKER --cucumberjson=reports/cucumber_report.json --cucumber-json-expanded --html=reports/general_report.html --self-contained-html
+docker run --rm $(env | cut -f1 -d= | sed 's/^/-e /') \
+  virtru/automated-test:latest pytest \
+  -v -s -n2 protect-and-track-website \
```

Comment:
@khvirtru This currently fails to find tests, and sets `$?` to 5. Running the tests from the ats3 folder via py.test does work. Any ideas what is wrong here? Are the tests shipped with the docker image?